minimisers/generalUtils/checkBounds.m

```diff
@@ -24,7 +24,7 @@
 
 fitWhichNbas = problem.fitBulkIn;
 fittingNbas = problem.nba(find(fitWhichNbas));
-fittingNbaConstr = problem.nbairs_constr(find(fitWhichNbas),:);
+fittingNbaConstr = problem.bulkIns_constr(find(fitWhichNbas),:);
```

Comment:
please remove the underscore

Reply:
I'll get on it, but we should be removing this file, as per #175.